bigchaindb/commands/utils.py

```diff
@@ -20,6 +20,10 @@ def start_rethinkdb():
 
     for line in proc.stdout:
         if line.startswith('Server ready'):
+            # FIXME: seems like tables are not ready when the server is ready,
```

Comment:
I just tried it, unfortunately it doesn't work because the database does not exits at all.
It seems to be equivalent to waiting on a non-existing database:
```
r.db('foobar').wait().run(c)
...
ReqlOpFailedError: Database `foobar` does not exist in:
r.db('foobar').wait()
^^^^^^^^^^^^^^       
```

Even if it's not the best solution, I'll keep a `time.sleep` for now. The right place to address the problem is in the workers, and to make them more solid on connecting to the database, so they reconnect when the connection to the DB is lost. This should be solved by #199 basically.